src/ReactiveUI/Bindings/Property/PropertyBinderImplementation.cs

```diff
@@ -596,30 +596,30 @@ object SetThenGet(object paramTarget, object paramValue, object[] paramParams)
                                                     signalInitialUpdate.Select(_ => true),
                                                     signalObservable);
 
-            var changeWithValues = somethingChanged.Select(isVm =>
+            var changeWithValues = somethingChanged.Select<bool, (bool isValid, object view, bool isViewModel)>(isVm =>
             {
                 if (!Reflection.TryGetValueForPropertyChain(out TVMProp vmValue, view.ViewModel, vmExpression.GetExpressionChain()) ||
                     !Reflection.TryGetValueForPropertyChain(out TVProp vValue, view, viewExpression.GetExpressionChain()))
                 {
-                    return null;
+                    return (false, null, false);
                 }
 
                 if (isVm)
                 {
                     if (!vmToViewConverter(vmValue, out TVProp vmAsView) || EqualityComparer<TVProp>.Default.Equals(vValue, vmAsView))
                     {
-                        return null;
+                        return (false, null, false);
                     }
 
-                    return Tuple.Create((object)vmAsView, isVm);
```

Comment:
This is just syntactic sugar to remove a Tuple instance for the more efficient ValueTuple.